### PR TITLE
Remove PackageToProjectFileAdder

### PR DESCRIPTION
**Pull request by wli3**
Add reference to project with version * is doing the same thing

### CODE REVIEW

**Comment by wli3** [comment]
https://github.com/dotnet/cli/issues/8365

File: test/Installer/Microsoft.DotNet.Cli.Msi.Tests/Microsoft.DotNet.Cli.Msi.Tests.csproj [hunk 1]
```diff
@@ -25,4 +25,8 @@
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
```
Comment (nguerrera) [inline test/Installer/Microsoft.DotNet.Cli.Msi.Tests/Microsoft.DotNet.Cli.Msi.Tests.csproj]: nit: revert (VS  bug adds this sometimes)

Comment (wli3) [reply test/Installer/Microsoft.DotNet.Cli.Msi.Tests/Microsoft.DotNet.Cli.Msi.Tests.csproj]: done, thanks. btw, that's a bug? And I just updated to latest, seems it is still there

Comment (nguerrera) [reply test/Installer/Microsoft.DotNet.Cli.Msi.Tests/Microsoft.DotNet.Cli.Msi.Tests.csproj]: well, i consider it a bug.

**Pull request merged**